neuralmonkey/model/sequence.py

```diff
@@ -82,6 +83,8 @@ def __init__(self,
             max_length: The maximum length of the sequences
             add_start_symbol: Includes <s> in the sequence
             add_end_symbol: Includes </s> in the sequence
+            multiply_embedding_mode: value="sqrt_depth" for tensor2tensor
```

Comment:
Tohle chce předělat, nejlíp oddědit transformer sekvenci od týhle sekvence a dávat ty vynásobený jako by nic.

Reply:
Nechal bych to na refaktor.

Reply:
přinejmenším tomu chybí docstring u `EmbeddedSequence`